app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerFragmentPresenter.kt

```diff
@@ -135,9 +194,17 @@ class QuestionPlayerFragmentPresenter @Inject constructor(
     val ephemeralQuestion = result.getOrThrow()
     // TODO(#497): Update this to properly link to question assets.
     val skillId = ephemeralQuestion.question.linkedSkillIdsList.firstOrNull() ?: ""
+
+    questionId = ephemeralQuestion.question.questionId
+    logger.e(
```

Comment:
Please remove this logger if not needed.

Reply:
Removed.